src/bp/core/logger/logger.ts

```diff
@@ -94,6 +94,11 @@ export class PersistedConsoleLogger implements Logger {
   }
 
   private print(level: LoggerLevel, message: string, metadata: any) {
+    if (typeof message !== 'string') {
+      metadata = message
+      message = '[warning] Message must be a string. If you want to log an object, use the second argument (metadata) '
```

Comment:
what about we make the message "(obj)" instead, since they obviously don't want to log a message and we make the metadata the object anyways